spec/views/curation_concerns/base/_form_progress.html.erb_spec.rb

```diff
@@ -64,4 +66,17 @@
       end
     end
   end
+
+  context "when the work has been saved before" do
+    before do
+      allow(work).to receive(:new_record?).and_return(false)
+      assign(:form, form)
+    end
+
+    let(:work) { stub_model(GenericWork, id: '456') }
```

Comment:
You shouldn't have had to move the assign.  It works from the inside out when it's resolving variables (`work` in this case).